src/toolkits/feature_engineering/word_counter.cpp

```diff
@@ -103,7 +109,7 @@ flexible_type word_counter_apply_with_manual(const flexible_type& input,
   // Tokenize all string inputs according to delimiters and to_lower options,
   // then accumulate counts and return as dictionary
   flexible_type output;
-  std::unordered_map<flexible_type, size_t> ret_count;
+  std::unordered_map<flexible_type, float> ret_count;
```

Comment:
flex_float is using double. why not using double? Otherwise, you will lose precision.